core/camel-jolokia/src/main/java/org/apache/camel/impl/jolokia/DefaultJolokiaHttpRequestHandlerFactory.java

```diff
@@ -36,17 +37,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class JolokiaHttpRequestHandlerSupport extends ServiceSupport implements StaticService {
+/**
+ * Default {@link org.apache.camel.console.DevConsoleRegistry}.
+ */
+@DevConsole("jolokia")
```

Comment:
This is wrong. @DevConsole is only for camel-console type of consoles.
You need to come up with another annotation such as JdkFactory.